### ISSUE TITLE
[Tutorial Update] Logging and Monitoring Basics

### ISSUE THREAD

**Issue by tomwanzek** [tensorflow/tensorflow#9526]
Please go to Stack Overflow for help and support:

http://stackoverflow.com/questions/tagged/tensorflow

If you open a GitHub issue, here is our policy:

1. It must be a bug or a feature request.
2. The form below must be filled out.

**Here's why we have that policy**: TensorFlow developers respond to issues. We want to focus on work that benefits the whole community, e.g., fixing bugs and adding features. Support only helps individuals. GitHub also notifies thousands of people when issues are filed. We want them to see you communicating an interesting problem, rather than being redirected to Stack Overflow.

------------------------

### System information

TensorFlow 1.1
n/a as purely documentation related

### Describe the problem

The currently published tutorial for [Logging and Monitoring](https://www.tensorflow.org/get_started/monitors) is based on functionality that has been identified as _deprecated_ (see i.p. the discussion in #7669 .)

I truly appreciate all the hard work 💯  put into moving the functionality of TF itself forward.

It would be most appreciated, if this tutorial could be updated to reflect suggested best practices when using `sessionRunHook` et al. to ease adoptabilty of TF.

### Source code / logs

n/a

**Comment by girving** [comment]
@martinwicke Seems like a reasonable request.  Assigning to you for learn triage.

**Comment by martinwicke** [comment]
@ispirmustafa I forget the last state of the Hooks tutorial. @sandersk, were you, or did you, already sign up to do this? 

@dr4b @wolffg FYI for docs planning.

**Comment by tomwanzek** [comment]
Any update on this? Thx in advance.

**Comment by martinwicke** [comment]
We're working on a docs update that does include a rewrite of this tutorial. It's not done yet.

**Comment by martinwicke** [comment]
@jugglerix feel free to close once the new Programmer's Guide section is up.

**Comment by jugglerix** [comment]
The current goal is to release the new Programmer's Guide in July.

**Comment by tomwanzek** [comment]
@jugglerix Is there an updated ETA for the new Programmer's Guide? Thx in advance.

**Comment by jugglerix** [comment]
The new Programmer's Guide will go out with V1.3.  However, we may add a
few additional chapters after 1.3 is released.

**Comment by tomwanzek** [comment]
Is there an update on the timing for this issue's resolution?
And a slightly more general contextual question, Is there a tracking reference to see which parts of the Programmer's Guide/Tutorials are current with Release 1.3 or still awaiting update as per @jugglerix above comment?

As always, thanks for the hard work!

**Comment by tensorflowbutler** [comment]
It has been 14 days with no activity and this issue has an assignee.Please update the label and/or status accordingly.

**Comment by jugglerix** [comment]
A few notes:

* The "Logging and Monitoring Basics with tf.contrib.learn" document was removed from tensorflow.org.
* The tutorial entitled "A Guide to TF Layers: Building a Convolutional Neural Network" demonstrates how to use SessionRunHook to set up logging during training.
